.github/workflows/buildtest.yml

```diff
@@ -178,14 +178,14 @@ jobs:
         config:
           - {name: "GCC",
              buildType: "Debug",
-             disable_gmm: "ON", # TOOD: enable again if GMM was fixed
+             disable_gmm: "OFF",
```

Comment:
Can you remove these lines and line 203, because we do not need them anymore?

Reply:
I'm reverting this change as GMM is apparently not fixed yet :)